src/main/java/com/bullhornsdk/data/model/entity/core/standard/Placement.java

```diff
@@ -1404,6 +1722,7 @@ public String toString() {
             ", commissions=" + commissions +
```

Comment:
@scriptom 
We missed adding all of the new fields to the 'toString()' and 'hashCode()' functions